src/build/zsh_completions.zig

```diff
@@ -47,50 +47,52 @@ fn writeZshCompletions(writer: anytype) !void {
         if (field.name[0] == '_') continue;
         try writer.writeAll("    \"--");
         try writer.writeAll(field.name);
-        try writer.writeAll("=-:::");
-
-        if (std.mem.startsWith(u8, field.name, "font-family"))
-            try writer.writeAll("_fonts")
-        else if (std.mem.eql(u8, "theme", field.name))
-            try writer.writeAll("_themes")
-        else if (std.mem.eql(u8, "working-directory", field.name))
-            try writer.writeAll("{_files -/}")
-        else if (field.type == Config.RepeatablePath)
-            try writer.writeAll("_files") // todo check if this is needed
-        else {
-            try writer.writeAll("(");
-            switch (@typeInfo(field.type)) {
-                .Bool => try writer.writeAll("true false"),
```

Comment:
It would be good to still handle the `.Bool` case in the switch like the suggestion above for fish and avoid the extra nesting. A different change would be needed roughly: 

- create a shared prefix ~`const equals_required="=-:::"` 
- add the prefix to all the cases where the previous the key has some required value

Reply:
Implemented your suggestion.